File: LeetCode-Problems/Easy/toLowerCase.java
//Java
class Solution {
    public String toLowerCase(String str) {
        int selected;
        String rString = "";
        for (int i = 0; i < str.length(); i++){
            selected = (int) str.charAt(i);
            if (selected < 91 && selected >= 65){
                rString += (char) (selected + 32);
            }
            else{
                rString += (char) selected;
            } 
        }
        return rString;
    }
}

File: LeetCode-Problems/Easy/toLowerCase.py
# Python
class Solution(object):
    def toLowerCase(self, str):
        """
        :type str: str
        :rtype: str
        """
        selected = 0
        rString = ""
        for x in range(0, len(str)):
            selected = ord(str[x:x+1])
            if (selected < 91 and selected >= 65):
                rString += chr(selected + 32)
            else:
                rString += chr(selected)
        return rString
